coop_memberspace/views/website/website_homepage.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
-<openerp><data noupdate="1">
+<odoo>
+    <data noupdate="1">
```

Comment:
```suggestion
```

Remove noupdate="1" here.
I don't get why we need it here, these are just qweb templates that are not meant to be customized